### ISSUE TITLE
Editorial Activity Report as a downloadable file

### ISSUE THREAD

**Issue by pmangahis** [pkp/pkp-lib#7428]
Journal Editors have requested to download the data from the Editorial Activity Report that is currently not possible to do through the User Interface

Ideally, they would like this report to also be available in CSV format.

**Comment by asmecher** [comment]
Sorry, @pmangahis, I'm not sure I follow this request; could you edit the entry to provide more detail? What verson of OJS are you using, what do you expect to be able to do, where do you get stuck, etc?

**Comment by NateWr** [comment]
In terms of UI/approach, we will probably follow whatever approach is adopted to publication stats report generation in https://github.com/pkp/pkp-lib/issues/7318.

**Comment by asmecher** [comment]
(Thanks for the details, @pmangahis!)

**Comment by NateWr** [comment]
@jonasraoni do you remember if the journal health report uses all the same data as the editorial activity report? Maybe we already have something that generates the CSV we need.

**Comment by jonasraoni** [comment]
@NateWr Yeah, as far as I remember, it has the same data, even the count of users per role.
The CSV generation is [inlined](https://github.com/pkp/pkp-lib/blob/main/classes/notification/managerDelegate/EditorialReportNotificationManager.inc.php#L68), so it will have to be extracted from there 😁
